### ISSUE TITLE
Come up with a name

### ISSUE THREAD

**Issue by dtenenba** [FredHutch/workflow-manager-hackathon#1]
If AWE is the name, we need to figure out what it stands for.

Or just come up with another name. The repo can be renamed. But the new name should be chosen before we publicize the project too much.

**Comment by bmcgough** [comment]
Well the obvious open source-y one is AWE is a Workflow Engine.

**Comment by dtenenba** [comment]
Yeah, but I feel like Cromwell is the workflow engine and we are just wrapping it and adding features.

**Comment by bmcgough** [comment]
AWE is a Workflow Environment?
